libs/langgraph/langgraph/utils.py

```diff
@@ -56,19 +56,21 @@ def __init__(
         recurse: bool = True,
         **kwargs: Any,
     ) -> None:
+        runnable_name = None
```

Comment:
why is this changing?

Reply:
otherwise there is an error about `self.name` not being defined, cause it's defined only in some branches without a default value

Reply:
then just add an `else`?